app/views/wikis/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Wiki's</h1>
+
+<div>
+  <ul>
+    <li><%= @wikis %></li>
```

Comment:
@bjblock - I originally had `<%= @wikis %> | Created by: <%= @wiki.user %>` but I kept running into the `undefined method 'user'` error. I attempted to defined `user` inside the index action of the Wiki controller but still didn't have any luck. How can I properly implement this here? Thanks!